nVolve/Features/Content/ContentView.swift

```diff
@@ -13,8 +13,14 @@ struct ContentView: View {
     @State var contentViewModel = ContentViewModel()
     @State private var position: MapCameraPosition = .automatic
     @StateObject private var viewModel = Markers()
-    @StateObject private var filterViewModel = FilterViewModel()
+    @StateObject private var filterViewModel: FilterViewModel
     @State private var showingFilters = false
+    
+    init() {
+            let contentViewModel = ContentViewModel()
+            _contentViewModel = State(wrappedValue: contentViewModel)
+            _filterViewModel = StateObject(wrappedValue: FilterViewModel(contentViewModel: contentViewModel))
+        }
```

Comment:
what does this do?

Reply:
This allows me to pass from contentViewModel to filterViewModel so I can use the events array in filterViewModel and apply filters.